.github/workflows/actions-ci.yml

```diff
@@ -462,6 +462,30 @@ jobs:
         run: |
           docker run -v "${{ github.workspace }}:/awslc" "gcc-4.8"
 
+  alpine-linux-x86:
+    needs: [sanity-test-run]
+    strategy:
+      fail-fast: false
+      matrix:
+        fips: [0, 1]
+        compiler: [
+          --build-arg CC=clang --build-arg CXX=clang++,
+          --build-arg CC=gcc --build-arg CXX=g++
+        ]
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build Docker Image
+        working-directory: .github/docker_images/alpine-linux
+        run: |
+          docker build -t alpine_linux ${{ matrix.compiler }} .
+      - name: Run tests
+        run: |
+          docker run -v "${{ github.workspace }}:/awslc" \
+          alpine_linux ${{ matrix.fips }}
```

Comment:
Actually instead of an entry script can we do it all here with the Docker run command?

Reply:
Good idea, implementing this now :)